frontend/components/BlogPreview.js

```diff
@@ -0,0 +1,91 @@
+//All necessary imports for this javascript
+import React from "react";
+import styles from "../styles/Blog.module.css";
+import Link from "next/link";
+import Chip from "@material-ui/core/Chip";
+import { useStyles } from "../data/BlogData";
+
+const BlogPreview = ({ individualBlogPreview }) => {
+  const classes = useStyles();
+
+  const execs = [
+    "/portraits/blog-authors/vivianw2021.jpg",
+    "/portraits/blog-authors/georgie2021.jpg",
+  ];
+  const { date, img, heading, subheading, blog_no, category } =
+    individualBlogPreview.fields;
+
+  const authors = individualBlogPreview.authors;
+
+  const imgUrl = "https:" + img.fields.file.url;
+
+  // contentfulDate = YYYY-MM-DD
+  // formattedDate = DD/MM/YYYY
+  const formattedDate = date.split("-").reverse().join("/");
+
+  return (
+    <div className={category}>
+      {/* Start of blog post preview */}
+      <div className={styles.blogPost}>
+        <Link href={`/media/blog/${blog_no}`}>
+          <div>
+            <div className={styles.darkOverlay} />
+            <div className={styles.previewRow}>
+              <div className={styles.previewContainerImg}>
+                <img
+                  className={styles.previewPic}
+                  src={`${imgUrl}`}
+                  alt="preview"
+                />
+              </div>
+              <div className={styles.previewContainerBlogInfo}>
+                <div className={styles.previewContainerBlogDetails}>
+                  <div className={styles.blogDetails}>
+                    <div className={styles.date}>{formattedDate}</div>
+                    <div className={styles.heading}>{heading}</div>
+                    <div className={styles.previewCategories}>
+                      {Object.keys(category).map((key) => (
+                        <Chip
```

Comment:
fix up responsiveness for the chips in the galaxy fold size

<img width="603" alt="image" src="https://user-images.githubusercontent.com/80564580/201578674-7f99d690-99cf-450b-87cb-1ef2b39b4dd5.png">